src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -352,7 +386,21 @@ public FilterResult calculateFilters(final M2FiltersArgumentCollection MTFAC, fi
         return filterResult;
     }
 
-    private int[] getIntArrayTumorField(final VariantContext vc, final String key) {
+    public FilterResult calculateMitochondrialFilters(MitochondrialFiltersArgumentCollection MTFAC, VariantContext vc) {
```

Comment:
In the subclass this method would be just be an override of `calcualteFilters()`